http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/db/LeoProfile.scala

```diff
@@ -313,6 +313,20 @@ private[leonardo] object LeoProfile extends MySQLProfile {
 
     implicit val instantSetParameter: SetParameter[Instant] =
       SetParameter.SetTimestamp.contramap(instant => java.sql.Timestamp.from(instant))
+
+    implicit val appUpgradeTableIdColumnType: BaseColumnType[UpdateAppTableId] =
+      MappedColumnType.base[UpdateAppTableId, Long](_.value, UpdateAppTableId.apply)
```

Comment:
nit, the table names (UpdateApp) are backward from the variable names (appUpdateJob); is there a reason?

Reply:
ah yeah, I started with `appUpgrade` and switched to `UpdateApp` for consistency with existing variables